src/xds/client.rs

```diff
@@ -274,16 +276,28 @@ impl AdsClient {
             .into_inner();
 
         info!("Stream established");
+        let (tx, rx) = oneshot::channel();
+        let mut initial_xds_tx = Some(tx);
+        let initial_xds_rx = Some(rx);
```

Comment:
Why do you wrap rx and tx

Reply:
The reason for wrapping `tx` is because I need to pass it into a function that's called in an infinite loop, but you're right I don't need to wrap `rx` here.